src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -37,6 +37,10 @@
           DestinationFolder="$(MicrosoftNetCoreAppRuntimePackNativeDir)"
           SkipUnchangedFiles="true" />
 
+    <!-- Run strip on the singlefilehost, in case it was built with symbols -->
+    <Exec Condition="'$(KeepNativeSymbols)' == 'true' and '$(HostOS)' != 'windows'"
```

Comment:
so this will strip symbols when `KeepNativeSymbols==true`? Maybe its easier to always strip?

Reply:
That would affect more than just the singlefilehost. I was trying to make this a narrow change.

Reply:
so `KeepNativeSymbols` is true only for th singlefilehost case?

Reply:
This is in the crossgen2 publish, so it only affects the singlefilehost when building crossgen.